Robust.Client/Audio/Midi/IMidiManager.cs

```diff
@@ -45,6 +45,11 @@ public interface IMidiManager
     /// </summary>
     RobustMidiEvent FromSequencerEvent(SequencerEvent midiEvent, uint tick);
 
+    /// <summary>
+    ///     Redirect for the MIDI library's soundfont checker
+    /// </summary>
+    public bool IsSoundFont(string filename);
```

Comment:
`public` is not really needed here
```suggestion
    bool IsSoundFont(string filename);
```